sources/api/netdog/src/cli/mod.rs

```diff
@@ -228,7 +228,9 @@ mod tests {
         let fake_file = tempfile::NamedTempFile::new().unwrap();
         let expected = "-net.ipv4.conf.eno1.rp_filter = 2\n";
         write_interface_sysctl(interface, &fake_file).unwrap();
-        assert_eq!(std::fs::read_to_string(&fake_file).unwrap(), expected);
+        assert!(std::fs::read_to_string(&fake_file)
+            .unwrap()
+            .starts_with(expected));
```

Comment:
This actually changes the semantics a bit no? We are checking for precisely the content listed above, did you need to make the `starts_with` change to get it to pass?

Reply:
systemd-networkd generates the correct line however, it also generates extra lines for ipv6 settings. Considering this test was looking for the first line i assumed keeping it the same would be preferable